buildkite/src/Command/Bench/Base.dhall

```diff
@@ -49,10 +51,13 @@ let Spec =
           { mode = PipelineMode.Type.PullRequest
           , size = Size.Perf
           , dependsOn =
-              DebianVersions.dependsOn
+              DebianVersions.dependsOnStep
```

Comment:
dependsOnStep is a inner function provide to a caller more control of depends step